static/app/components/replays/diff/replayDiffChooser.tsx

```diff
@@ -80,26 +77,6 @@ export default function ReplayDiffChooser({
           </TabPanels.Item>
         </TabPanels>
       </Flex>
-    </TabStateProvider>
-  );
-}
-
-// TODO(ryan953): put this into static/app/components/tabs/index.tsx near `<Tabs>`
-function TabStateProvider<T extends string | number>({
```

Comment:
why was this added then deleted🤔

Reply:
it's more UI focused, i will followup and make the Modal better, but not in this PR